playbooks/common/openshift-cluster/openshift_logging.yml

```diff
@@ -0,0 +1,4 @@
+- name: OpenShift Aggregated Logging
+  hosts: all
```

Comment:
This probably shouldn't be `hosts: all`. I would suggest using `hosts: oo_first_master`. It might also be good to add a "wrapper playbook" in `playbooks/byo/openshift-cluster` that looks something like the following:

```
---
- include: ../../common/openshift-cluster/verify_ansible_version.yml

- name: Create initial host groups for localhost
  hosts: localhost
  connection: local
  become: no
  gather_facts: no
  tags:
  - always
  tasks:
  - include_vars: ../../byo/openshift-cluster/cluster_hosts.yml
  - add_host:
      name: "{{ item }}"
      groups: l_oo_all_hosts
    with_items: "{{ g_all_hosts | default([]) }}"

- name: Create initial host groups for all hosts
  hosts: l_oo_all_hosts
  gather_facts: no
  tags:
  - always
  tasks:
  - include_vars: ../../byo/openshift-cluster/cluster_hosts.yml

- include: ../../common/openshift-cluster/openshift_logging.yml
  vars:
    openshift_cluster_id: "{{ cluster_id | default('default') }}"
    openshift_debug_level: "{{ debug_level | default(2) }}"
    openshift_deployment_type: "{{ deployment_type }}"
```